Resources/doc/annotations/cache.rst

```diff
@@ -99,6 +99,7 @@ Annotation                                                              Response
 ``@Cache(expires="tomorrow")``                                          ``$response->setExpires()``
 ``@Cache(smaxage="15")``                                                ``$response->setSharedMaxAge()``
 ``@Cache(maxage="15")``                                                 ``$response->setMaxAge()``
+``@Cache(maxstale="15")``                                               ``$response->headers->addCacheControlDirective('max-stale', 15)``
```

Comment:
We should probably add a note before or after this table to explain that one can use text as well.